### ISSUE TITLE
Firefox can't establish a websocket connection while Chrome can

### ISSUE THREAD

**Issue by wjkang** [websockets/ws#1450]
- [x] I've searched for any related issues and avoided creating a duplicate issue.

#### Description

Firefox can't establish a websocket connection while Chrome can

code like
```js
const config = {
    port: 3002,
    perMessageDeflate: {
        zlibDeflateOptions: { // See zlib defaults.
            chunkSize: 1024,
            memLevel: 7,
            level: 3,
        },
        zlibInflateOptions: {
            chunkSize: 10 * 1024
        },
        // Other options settable:
        clientNoContextTakeover: true, // Defaults to negotiated value.
        serverNoContextTakeover: true, // Defaults to negotiated value.
        clientMaxWindowBits: 10,       // Defaults to negotiated value.
        serverMaxWindowBits: 10,       // Defaults to negotiated value.
        // Below options specified as default values.
        concurrencyLimit: 10,          // Limits zlib concurrency for perf.
        threshold: 1024,               // Size (in bytes) below which messages
        // should not be compressed.
    }
}
const server = new WebSocket.Server(config);
server.on('connection', (client, req) => {
    ...
});
```
[real code](https://github.com/wjkang/lazy-mock/blob/master/src/lib/EasySocket.js)

#### Reproducible in:

version:
Node.js version(s):v9.0.0
OS version(s):window 10，Linux Ubuntu 16

#### Steps to reproduce:

[online demo](http://jaycewu.coding.me/easy-socket-chat/#/login)

### Actual result:
Firefox can't establish a websocket connection

![image](https://user-images.githubusercontent.com/9456046/46189171-e1882a80-c31f-11e8-9b40-79cc6a9544ae.png)

**Comment by lpinca** [comment]
See discussion in #1446.
TLDR: do not set the `clientMaxWindowBits` option.

**Comment by wjkang** [comment]
thank you,it works
